infra/modules/monitoring/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "ecs_name" {
+  type        = string
+  description = "Target ECS cluster name"
+}
+
+variable "cpu_threshold" {
```

Comment:
let's base the name of these threshold variables on the name of the alarms. in other words, this should be something like `high_cpu_usage_alert_threshold`

Reply:
makes sense

Reply:
done